src/api/upgrade_storage/mod.rs

```diff
@@ -12,41 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use alloc::vec::Vec;
 use persistent_store::StorageResult;
 
 pub(crate) mod helper;
 
 /// Accessors to storage locations used for upgrading from a CTAP command.
 pub trait UpgradeStorage {
-    /// Reads a slice of the partition, if within bounds.
+    /// Writes the given data as part of an upgrade.
```

Comment:
nit: We could say "Processes" instead of "Writes".

Reply:
Done